assets/js/modules/analytics-4/components/setup/SetupMain.js

```diff
@@ -116,25 +127,35 @@ export default function SetupMain( { finishSetup } ) {
 	}
 
 	return (
-		<div className="googlesitekit-setup-module googlesitekit-setup-module--analytics">
-			<div className="googlesitekit-setup-module__step">
-				<div className="googlesitekit-setup-module__logo">
-					<AnalyticsIcon width="40" height="40" />
-				</div>
+		<Fragment>
+			<div className="googlesitekit-setup-module googlesitekit-setup-module--analytics">
+				<div className="googlesitekit-setup-module__step">
+					<div className="googlesitekit-setup-module__logo">
+						<AnalyticsIcon width="40" height="40" />
+					</div>
 
-				<Typography
-					as="h3"
-					className="googlesitekit-setup-module__title"
-					size="small"
-					type="headline"
-				>
-					{ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
-				</Typography>
-			</div>
-			<div className="googlesitekit-setup-module__step">
-				{ viewComponent }
+					<Typography
+						as="h3"
+						className="googlesitekit-setup-module__title"
+						size="small"
+						type="headline"
+					>
+						{ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
+					</Typography>
+				</div>
+				<div className="googlesitekit-setup-module__step">
+					{ viewComponent }
+				</div>
 			</div>
-		</div>
+			{ showSearchConsoleSetupSuccessToast && (
+				<ToastNotice
+					title="Search Console was successfully set up"
```

Comment:
Please remember to translate user-facing strings:
```suggestion
					title={ __(
						'Search Console was successfully set up',
						'google-site-kit'
					) }
```